app/src/Components/Event/EventOptions.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Shamelessly duplicated from ProjectOptions.tsx
+ */
+import React from 'react'
+import styled from 'styled-components/native'
+import underDevelopmentAlert from '../../Utils/UnderDevelopmentAlert'
+
+const EventOptionsView = styled.View`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 23px 10px;
+`
+
+const EventOptionsText = styled.Text`
+  font-size: 18px;
+`
+
+const EventOptionsTouch = styled.TouchableOpacity``
+
+const EventOptions = () => {
+  return (
+    <EventOptionsView>
+      <EventOptionsTouch onPress={underDevelopmentAlert}>
+        <EventOptionsText>Past</EventOptionsText>
+      </EventOptionsTouch>
+      <EventOptionsTouch onPress={underDevelopmentAlert}>
```

Comment:
The current events list is actually upcoming events (i.e. we have implemented this), so this could instead navigate to that screen and/or be bolded to show it's currently selected?